engines/engine.cpp

```diff
@@ -210,6 +210,9 @@ void initCommonGFX() {
 
 		if (gameDomain->contains("filtering"))
 			g_system->setFeatureState(OSystem::kFeatureFilteringMode, ConfMan.getBool("filtering"));
+
+		if (gameDomain->contains("stretch_mode"))
+			g_system->setStretchMode(ConfMan.get("stretch_mode").c_str());
```

Comment:
It would be good if this could be merged seperately from the rest of this pull request, since it is a good fix anyway and I think this may be the fix for a problem reported on the forum a few days back:
http://forums.scummvm.org/viewtopic.php?t=14732

Reply:
Oops! This is an oversight from myself when I added the feature. But it is indeed unrelated to this PR. I will commit that change this evening when I am back home (unless somebody does it before me).

Reply:
@criezy : Happy for you to commit it later. Though @eriktorbjorn will have to rebase the PR at that point to remove this commit...